raiden_libs/messages/json_schema.py

```diff
@@ -39,7 +39,7 @@
     'type': 'object',
     'required': [
         'channel_identifier', 'nonce', 'transferred_amount', 'locksroot',
```

Comment:
Can you make a new line for each required field, this is quite hard to read.

Reply:
nooooooo.. what a waste of  vertical space

But since it's python code and I am lazy I'll set that `required` list with a function ;)

Reply:
From *The Zen of Python*
> Readability counts.




🤓